Makefile

```diff
@@ -12,6 +12,19 @@ COMPILER ?= gcc
 # If DEBUG_BUILD is 1, compile with DEBUG_ROM defined
 DEBUG_BUILD ?= 1
 
+# Valid compression algorithms are yaz and lzo
+COMPRESSION ?= yaz
+
+ifeq ($(COMPRESSION),lzo)
+  CFLAGS += -DCOMPRESSION_LZO
+  CPPFLAGS += -DCOMPRESSION_LZO
+endif
+
+ifeq ($(COMPRESSION),yaz)
+  CFLAGS += -DCOMPRESSION_YAZ
+  CPPFLAGS += -DCOMPRESSION_YAZ
+endif
+
```

Comment:
I was thinking, we should probably add something in the readme to explain the different compression algorithms with their advantages and their drawbacks

Reply:
z64enc has a nice writeup in their readme https://github.com/z64me/z64enc

Reply:
maybe add this link to the readme then? I'd like to have it in the readme for people searching informations on this